marshmallow_sqlalchemy/schema.py

```diff
@@ -137,11 +137,14 @@ class Meta:
     """
     OPTIONS_CLASS = ModelSchemaOpts
 
+    @property
+    def session(self):
+        return self._session or self.opts.sqla_session
```

Comment:
This is a breaking change since `session` was a public, write-able attribute. Adding a `setter` should make this backwards-compatible.